### ISSUE TITLE
Check accessibility for svg icons

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#1238]
Hello Alan.

These are the three that fail accessibility. Chemistry has the Link to Learning; A&P will have the homeostatic imbalances one for you to test. 

Note: This is for testing the lighthouse app to see how badly this flags 

<img width="409" alt="Screen Shot 2019-07-23 at 4 11 01 PM" src="https://user-images.githubusercontent.com/11299125/61747896-8fa80980-ad64-11e9-95f1-efcdc53c940b.png">
<img width="394" alt="Screen Shot 2019-07-23 at 4 11 05 PM" src="https://user-images.githubusercontent.com/11299125/61747898-8fa80980-ad64-11e9-89b8-548f2994429c.png">
<img width="388" alt="Screen Shot 2019-07-23 at 4 11 10 PM" src="https://user-images.githubusercontent.com/11299125/61747899-8fa80980-ad64-11e9-9552-0ab0eb7c3c5b.png">
<img width="397" alt="Screen Shot 2019-07-23 at 4 11 14 PM" src="https://user-images.githubusercontent.com/11299125/61747900-8fa80980-ad64-11e9-9e21-be6e331d9c66.png">
<img width="387" alt="Screen Shot 2019-07-23 at 4 11 17 PM" src="https://user-images.githubusercontent.com/11299125/61747901-8fa80980-ad64-11e9-9930-31e2701a1a21.png">
<img width="386" alt="Screen Shot 2019-07-23 at 4 11 22 PM" src="https://user-images.githubusercontent.com/11299125/61747902-8fa80980-ad64-11e9-8435-f12621cae1d1.png">
